core/plugins/org.polarsys.capella.core.platform.sirius.sirius.validation/src/org/polarsys/capella/core/platform/sirius/sirius/validation/ddiagram/LinkDescription.java

```diff
@@ -0,0 +1,60 @@
+package org.polarsys.capella.core.platform.sirius.sirius.validation.ddiagram;
+
+import org.eclipse.emf.ecore.EObject;
+import org.xml.sax.Attributes;
+
+public class LinkDescription {
+	private final String name;
+	private final String href;
+	private EObject targetElement;
+	private Attributes attributes;
+	
+	public LinkDescription(String name, String href, EObject targetElement, Attributes attributes) {
+		this.name = name;
+		this.href = href;
+		this.targetElement = targetElement;
+		this.attributes = attributes;
+	}
+	
+	public String getName() {
+		return name;
+	}
+	
+	public String getHref() {
+		return href;
+	}
+	
+	public EObject getTargetElement() {
+		return targetElement;
+	}
+	
+	public void setTargetElement(EObject targetElement) {
```

Comment:
Are setters actually used? If not, remove them (prefer immutable objects when possible)